omniduct/databases/hiveserver2.py

```diff
@@ -425,6 +427,7 @@ def _run_in_hivecli(self, cmd):
         # Turn hive command into quotable string.
         double_escaped = re.sub('\\' * 2, '\\' * 4, cmd)
         sys_cmd = 'hive -e "{0}"'.format(re.sub('"', '\\"', double_escaped))
+        sys_cmd = sys_cmd.replace('`', r'\\\`')
```

Comment:
As per my earlier code suggestion, I'd move the substitution before the double escaping, to keep the number of explicitly typed backslashes to a minimum.

Reply:
Since we need 3 literal slashes before the backtick it looks like I need to have this after the `double_escaped`.